src/components/Layout/Layout.tsx

```diff
@@ -8,21 +8,29 @@ import { LeftSideBar } from 'src/components/StaticQuerySidebar';
 import GlobalLoading from '../GlobalLoading/GlobalLoading';
 import { Footer } from '../Footer';
 
-const Layout: FC<{ isExtraWide?: boolean }> = ({ children, isExtraWide = false }) => {
+const Layout: FC<{ isExtraWide?: boolean; noSidebar?: boolean }> = ({
+  children,
+  isExtraWide = false,
+  noSidebar = false,
+}) => {
   return (
     <GlobalLoading>
       <Header />
-      <LeftSideBar />
+      {noSidebar ? null : <LeftSideBar />}
```

Comment:
Minor, I would have gone for something like the below, so that the conditions are inverted and we always think of the truthy case first.

~~~js
# At the top flip the condition
const showSidebar = !noSidebar;
~~~

~~~jsx
{showSidebar && <LeftSidebar/>}
// ...
... className={ showSidebar ? cn('...') : null }
~~~